packages/edit-site/src/components/block-editor/use-site-editor-settings.js

```diff
@@ -97,8 +97,6 @@ export default function useSiteEditorSettings() {
 		keepCaretInsideBlock,
 		canvasMode,
 		settings,
-		postType,
-		postId,
```

Comment:
Tiny nit: these are both still returned by the `useSelect` even though they're not retrieved in this destructuring. Is it worth removing them from within the `useSelect` callback, too (lines 133 and 134), since the corresponding lines are now handled in `useSiteEditorSettings`?